k8s/airflow-operator/manifest/application.yaml.template

```diff
@@ -0,0 +1,40 @@
+    marketplace.cloud.google.com/deploy-info: '{partner_id: "click-to-deploy-containers", product_id: "airflow-operator", parnter_name: "Google Click to Deploy"}'
+  labels:
+    app.kubernetes.io/name: "$name"
+spec:
+  type: Airflow Operator
+  componentKinds:
+  - group: apps/v1beta2
+    kind: Deployment
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: "$name"
+  version: latest
+  description: |-
+    Airflow Operator enables specifying and running Apache Airflow on Kubernetes.
+
+    # Support
+    Google does not offer support for this solution. However, community support is available on
+    [Stack Overflow](https://stackoverflow.com/questions/tagged/airflow/).
+    Additional community support is available on [Airflow gitter](https://gitter.im/apache/incubator-airflow).
+  maintainers:
+  - name: Google Click to Deploy
+    url: https://cloud.google.com/solutions/#click-to-deploy
+  links:
+  - description: 'User Guide: Google Click to Deploy Airflow Operator'
+    url: https://console.cloud.google.com/code/develop/browse/google-click-to-deploy/master/k8s/airflow-operator/README.md?project=k8s-marketplace-eap
```

Comment:
things have changed a bit since we already launched. Please use github repo directly here

Reply:
done.